src/main/java/edu/cornell/kfs/fp/batch/service/impl/ProcurementCardLoadFlatTransactionsServiceImpl.java

```diff
@@ -79,8 +79,8 @@ public boolean loadProcurementCardFile(String fileName) {
             LOG.error("error while getting file bytes:  " + e.getMessage(), e);
             throw new RuntimeException("Error encountered while attempting to get file bytes: " + e.getMessage(), e);
         } catch (ParseException e) {
-            LOG.error(ERROR_PREFIX + e.getMessage());
-            throw new RuntimeException(ERROR_PREFIX + e.getMessage(), e);
+            LOG.error(ERROR_PREFIX  + fileName + e.getMessage());
+            throw new RuntimeException(ERROR_PREFIX + fileName + e.getMessage(), e);
```

Comment:
With these code review changes, you will also notice a space was added in the above error messages after the file name. It was needed for readability.